homeassistant/components/simplisafe/__init__.py

```diff
@@ -389,16 +407,26 @@ async def async_update_system(system):
 class SimpliSafeEntity(CoordinatorEntity):
     """Define a base SimpliSafe entity."""
 
-    def __init__(self, simplisafe, system, name, *, serial=None):
+    def __init__(
+        self,
+        simplisafe: SimpliSafe,
+        system: System,
+        name: str,
+        *,
+        serial: str | None = None,
+    ) -> None:
         """Initialize."""
-        super().__init__(simplisafe.coordinator)
+        coordinator = cast(DataUpdateCoordinator, simplisafe.coordinator)
+        super().__init__(coordinator)
 
         if serial:
             self._serial = serial
         else:
             self._serial = system.serial
 
-        self._attr_extra_state_attributes = {ATTR_SYSTEM_ID: system.system_id}
+        self._attr_extra_state_attributes: MutableMapping[str, Any] = {
```

Comment:
Same as with the other PRs, we should not override the typing here.
I know from other comments this is a problem, I'm willing to take a look. But we should not merge this in.

We need to resolve this first.